subsys/net/lib/zperf/zperf_tcp_receiver.c

```diff
@@ -254,52 +274,62 @@ static void tcp_server_session(void)
 			}
 
 			switch (i) {
-			case SOCK_ID_IPV4_LISTEN:
-				addr = &addr_ipv4;
-				__fallthrough;
-			case SOCK_ID_IPV6_LISTEN:{
-				int sock = zsock_accept(fds[i].fd, addr, &addrlen);
+			case SOCK_ID_IPV4_LISTEN ... SOCK_ID_IPV6_LISTEN: {
```

Comment:
I'm actually not sure if we should use GNU C extensions, AFAIK we should strive for portability and not rely on specific compiler features. @carlescufi Is that correct, or am I exaggerating?

Reply:
This is a valid point, however also clang supports case ranges, and I've seen some place in the codebase where they are used.
I don't have, however, a strong objection to remove them, in the end case(s) are just if(s).

Let me know your preference

Reply:
IMHO these extensions should be avoided unless there is no other alternative. Here we have only couple of values to check so I would only use standard C in this case.

Reply:
fixed